src/psconvert.c

```diff
@@ -731,7 +731,7 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
 	GMT_Usage (API, -2, "Note: Shows the gdal_translate command, in case you want to use this program "
 		"to create a geoTIFF file.");
 	GMT_Usage (API, 1, "\n-W[+a<mode>[<alt]][+c][+f<minfade>/<maxfade>][+g][+k][+l<lodmin>/<lodmax>][+n<name>][+o<folder>][+t<title>][+u<URL>]");
-	GMT_Usage (API, -2, "Write a ESRI type world file suitable to make (e.g.,) .tif files "
+	GMT_Usage (API, -2, "Write an ESRI type world file suitable to make e.g. .tif files "
```

Comment:
```suggestion
	GMT_Usage (API, -2, "Write an ESRI type world file suitable to make .tif files "
```
I think the e.g., here detracts from the clarity.